rules/rules-reviewed/eap8/eap7/tests/jakarta-faces.windup.test.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruletest id="jakarta-faces-tests"
+          xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <testDataPath>data/jakarta-faces</testDataPath>
+    <rulePath>../jakarta-faces.windup.xml</rulePath>
+    <ruleset>
+        <rules>
+            <rule id="jakarta-faces-00001-test">
+                <when>
+                    <not>
```

Comment:
by convention we specify the number of times we expect the rule to fire in the test here usually